Lombiq.Hosting.Azure.ApplicationInsights/Extensions/ApplicationInsightsInitializerExtensions.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Lombiq.Hosting.Azure.ApplicationInsights;
+using Lombiq.Hosting.Azure.ApplicationInsights.Services;
+using Lombiq.Hosting.Azure.ApplicationInsights.TelemetryInitializers;
+using Microsoft.ApplicationInsights.AspNetCore.Extensions;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DependencyCollector;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.QuickPulse;
+using Microsoft.Extensions.Configuration;
+using System.Linq;
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+public static class ApplicationInsightsInitializerExtensions
+{
+    /// <summary>
+    /// Use this extension method to initialize Application Insights. Should be used in the application Program.cs file.
+    /// </summary>
+    public static IServiceCollection AddOrchardCoreApplicationInsightsTelemetry(
```

Comment:
Since the module is also needed on all tenants, this could also enable it with `orchardCoreBuilder.AddTenantFeatures()`.

Reply:
This isn't addressed.

Reply:
I thought you meant this:
https://github.com/Lombiq/DotNest-Tenants-Core/pull/189/files/d5ed4600708daa397e48790a720f2cd0fe9f6c42..d728cf69139984ce0efc044438eedd9504708186#diff-23d697d652de712ccf9d2c42af3ecd2d4a59e0b70c1385b48d959d00def2d2f1R46

Then I obviously don't know what you mean.

Reply:
You can enable the module for all tenants here in this method, so consumers don't have to do that since it's always necessary anyway.

Reply:
oh okay.